app/helpers/audit_helper.rb

```diff
@@ -52,8 +52,6 @@ def activity(type_of_activity, args = nil)
       "Application withdrawn"
     when "closed"
       "Application closed"
```

Comment:
This `#activity` helper accepts a key and returns a string...ie exactly what `I18n.t` does 😆. Future refactor to move all this to locale files 😁